### PR TITLE
ENG-7479: Configurable DR overflow dir

### PR DESCRIPTION
**Pull request by nshi**

### CODE REVIEW

File: src/frontend/org/voltdb/utils/CatalogUtil.java [hunk 1]
```diff
@@ -1434,6 +1440,27 @@ public static File getExportOverflow(PathsType.Exportoverflow paths, File voltDb
 
     }
 
+    public static File getDROverflow(PathsType.Droverflow paths, File voltDbRoot) {
```
Comment (afrogers) [inline src/frontend/org/voltdb/utils/CatalogUtil.java]: So DR overflow is in reality required to be absolute. Seems slightly odd to have a test utility accepting a relative one. I guess it's convenient, though.

**Comment by afrogers** [comment]
Review complete

File: src/frontend/org/voltdb/utils/CatalogUtil.java (hunk 1 above)
Comment (nshi) [inline src/frontend/org/voltdb/utils/CatalogUtil.java]: Not sure I follow. This method transforms a relative DR overflow path to be an absolute path under voltroot. You can specify relative paths in the deployment file.

Comment (afrogers) [inline src/frontend/org/voltdb/utils/CatalogUtil.java]: So you can... looking back, not sure I follow my own reasoning. Merge away.

On Tue, Mar 31, 2015 at 9:59 PM, Ning Shi <notifications@github.com> wrote:

> In src/frontend/org/voltdb/utils/CatalogUtil.java
> <https://github.com/VoltDB/voltdb/pull/2189#discussion_r27539788>:
>
> > @@ -1434,6 +1440,27 @@ public static File getExportOverflow(PathsType.Exportoverflow paths, File voltDb
> >
> >      }
> >
> > +    public static File getDROverflow(PathsType.Droverflow paths, File voltDbRoot) {
>
> Not sure I follow. This method transforms a relative DR overflow path to
> be an absolute path under voltroot. You can specify relative paths in the
> deployment file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/VoltDB/voltdb/pull/2189/files#r27539788>.
>



-- 

*[image: VoltDBLogoEmail]* <http://www.voltdb.com/>
Alex Rogers, Software Engineer
*VoltDB, Inc.*
*E: *arogers@voltdb.com

*voltdb.com* <http://www.voltdb.com/>
Follow VoltDB [image: twitter] <https://twitter.com/VoltDB> [image:
facebook] <https://www.facebook.com/VoltDB> [image: linkedin]
<https://www.linkedin.com/company/voltdb>

<http://www.linkedin.com/company/voltdb>

**Pull request merged**